src/test/resources/pages/NJIfeedbackSessionRemind.html

```diff
@@ -385,7 +385,7 @@ <h5 class="col-sm-2">
                             <div data-original-title="Select this option to automatically send an email to students to notify them when the session is open for submission." class="col-sm-3" title="" data-toggle="tooltip" data-placement="top">
                                 <div class="checkbox">
                                     <label>Session opening
-                                        reminder</label> <input checked="checked" name="sendreminderemail" id="sendreminderemail_open" value="FEEDBACK_OPENING" type="checkbox" disabled="disabled">
+                                        reminder</label> <input checked="checked" name="sendreminderemail" id="sendreminderemail_open" value="FEEDBACK_OPENING" disabled="disabled" type="checkbox">
```

Comment:
This change does not seem related to this issue. Does it cause a test failure if unchanged?